### DISCUSSION TITLE
[META\FLUFF] What systems do flycast currently support?

### DISCUSSION THREAD

**Discussion by cinnamonmatexfce** [flyinghead/flycast#1784] [General]
- Dreamcast (our dear beloved 💖)
 - NAOMI
 - NAOMI 2
 - [Sega System SP (Spider) BIOS](http://adb.arcadeitalia.net/dettaglio_mame.php?game_name=segasp&arcade_only=0&autosearch=1&lang=en) (as found in [here](https://github.com/flyinghead/flycast/releases/tag/v2.2))

Are these all? Or am I missing some?

Thanks

**Comment by flyinghead** [comment]
Atomiswave
